### PR TITLE
Update and rename N_0x5e657ef1099edd65.md to DoesLatestBriefStringExi…

### PR DESCRIPTION
**Pull request by freedy69**
…st.md

Updated the description and the file name (didn't have the name changed in my last commit)

### CODE REVIEW

File: GRAPHICS/DoesLatestBriefStringExist.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+ns: GRAPHICS
```
Comment (AvarianKnight) [inline GRAPHICS/DoesLatestBriefStringExist.md:2]: You're missing the aliases, which should include the previous native name(s), you can see [here](https://github.com/citizenfx/natives/edit/master/NETWORK/NetworkUseHighPrecisionBlending.md) for an example, though note you don't have to include the 'Old Name' tidbit it's just leftovers from a NativeDB import

File: GRAPHICS/DoesLatestBriefStringExist.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+---
+## _0x5E657EF1099EDD65
```
Comment (AvarianKnight) [inline GRAPHICS/DoesLatestBriefStringExist.md:4]: You forgot to rename the native here

File: GRAPHICS/DoesLatestBriefStringExist.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+```c
+// 0x5E657EF1099EDD65 0x91A081A1
+BOOL DOES_LATEST_BRIEF_STRING_EXIST(int briefType);
+```
+
+```
```
Comment (AvarianKnight) [inline GRAPHICS/DoesLatestBriefStringExist.md:11]: Code blocks should only be used for native db imports

File: GRAPHICS/DoesLatestBriefStringExist.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+Checks if brief text is available.
+
+Brief types:
+0 - Dialogue Brief
+1 - Help Text Brief
+2 - Mission Objective Brief
+```
+
+```c
```
Comment (AvarianKnight) [inline GRAPHICS/DoesLatestBriefStringExist.md:20]: This example wouldn't really be applicable for others, and doesn't provide much use for what its actually used for

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

File: GRAPHICS/DoesLatestBriefStringExist.md (hunk 4 above)
Comment (freedy69) [reply GRAPHICS/DoesLatestBriefStringExist.md:20]: ah, can i somehow edit this pull request or do i have to make a new one?

**Pull request closed without merging**